packages/dappmanager/src/modules/chains/getChainDriverName.ts

```diff
@@ -18,5 +18,6 @@ const knownChains: { [dnpName: string]: ChainDriver } = {
   "bitcoin.dnp.dappnode.eth": "bitcoin",
   "monero.dnp.dappnode.eth": "monero",
   "prysm.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm",
-  "prysm-prater.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm"
+  "prysm-prater.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm",
+  "handshake.dnp.dappnode.eth": "handshake"
```

Comment:
So this mapping of package name to driver was meant to boostrap the feature. If you include a driver property in the manifest of your package it will have the same effect.

Reply:
Done.